website/src/components/ComponentHeadline.astro

```diff
@@ -1,7 +1,17 @@
 ---
+import { fragmentLinkHeaderOffset } from '../layouts/base/header/headerConstants';
+interface Props {
+    title: string;
+}
 
+const { title } = Astro.props;
+
+const fragmentId = title.toLowerCase().replace(/ /g, '-');
 ---
 
-<h2 class='font-bold'>
-    <slot />
+<h2 class={`font-bold ${fragmentLinkHeaderOffset} `} id={fragmentId}>
```

Comment:
```suggestion
<h2 class={`font-bold scroll-mt-20`} id={fragmentId}>
```
I think this will do the same. We could also put this into the headerConstants, to see that it has the same height as the header. But maybe we could also leave it here.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-margin-top